README.md

```diff
@@ -195,6 +195,16 @@ A. Local respoitories are not copied to Helm v3. You therefore need to remove al
 required using `<helm3> repo add`. This is a necessary refresh to align references for Helm v3 and remove the conflict. It is worthwhile to also refresh the
 repository list afterwards: `<helm3> repo update`. You should then be able to run the chart dependency update command successfully.
 
+## Frequently Asked Questions
+
+***Q. Have a plan to support Helm v2 release migration in batch mode?***
+
+A. We do not want to extend `covert` subcmd, but you can perform batch migration of releases using a command as follows:
```

Comment:
We do not want to extend `covert` subcmd, but you can perform batch migration of releases using a command as follows: --> You can perform batch migration of releases using a command as follows: